test/command_callback/test_hclfmt_command_callback.vader

```diff
@@ -0,0 +1,8 @@
+Before:
+  call ale#assert#SetUpLinterTest('hcl', 'hclfmt')
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(The default command should be correct):
+  AssertLinter 'hclfmt', ale#Escape('hclfmt') . ' -w' . ' %t'
```

Comment:
The test fails because you escape the executable name here, but not in your linter file.